sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -125,7 +129,7 @@ case class DescribeFunctionCommand(
 
           if (isExtended) {
             result :+
-              Row(s"Extended Usage:\n${replaceFunctionName(info.getExtended, info.getName)}")
+              Row(s"Extended Usage:${replaceFunctionName(info.getExtended, info.getName)}")
```

Comment:
I also just saw that you removed newline here, so please disregard my previous comment on adding one before the extended description